From 7295e71daa61052880e80e437a99f5e3e61789e6 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Luc=20Noval=C3=A8s?= <luc.novales@free.fr>
Date: Sun, 8 Apr 2018 17:16:40 +0200
Subject: [PATCH 1/5] This command line is more universal, it works too in
 foreign langages

Without forcing langage in english command fails with error "keytool error : java.util.IllegalFormatConversionException: d != java.lang.String"

Change-Id: Iab005cb05457b697e6f66dd12c85a459214f8575
---
 pages/meta/verifying_builds.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pages/meta/verifying_builds.md b/pages/meta/verifying_builds.md
index 40f225c..42f4a2a 100644
--- a/pages/meta/verifying_builds.md
+++ b/pages/meta/verifying_builds.md
@@ -8,7 +8,7 @@ permalink: verifying-builds.html
 All official builds from LineageOS are signed with our private keys. You can verify a build has been signed with our keys by running: 
 
 ```
-keytool -list -printcert -jarfile lineage-build-signed.zip
+keytool -J-Duser.language=en -list -printcert -jarfile lineage-build-signed.zip
 ```
 
 The resulting fingerprints are listed as follows: 
-- 
2.17.1

